test/e2e/fpga/fpga.go

```diff
@@ -32,7 +34,7 @@ import (
 
 const (
 	pluginDeployScript  = "scripts/deploy-fpgaplugin.sh"
```

Comment:
can we get rid of `pluginDeployScript` too?

Reply:
I think this would be the next step. This PR is about webhook deployment as far as I understood.

Reply:
OK, true. Another PR is fine